src/Controller/SelfServiceController.php

```diff
@@ -92,6 +92,17 @@ public function index()
                 $customBarcodeFound = true;
                 $attributeId = 0;
             }
+            else {
+                $prefixbarcodewithweight = "27"; //Mario: an welcher globalen Stelle willst du das hinterlegen?
```

Comment:
das bitte in die app_config.php unterhalb von `define('MAX_CART_PRODUCT_AMOUNT', 250);`
also
```
define('BARCODE_WITH_WEIGHT_PREFIX', 27);
```

darauf kannst du dann global über `BARCODE_WITH_WEIGHT_PREFIX` zugreifen